app/models/incomeSourceDetails/BusinessAddressModel.scala

```diff
@@ -18,16 +18,10 @@ package models.incomeSourceDetails
 
 import play.api.libs.json.{Json, OFormat}
 
-case class Address(lines: Seq[String], postcode: Option[String]) {
-  override def toString: String = s"${lines.mkString(", ")}${postcode.map(stringValue => s", $stringValue").getOrElse("")}"
-}
-
-case class BusinessAddressModel(auditRef: String,
-                                address: Address)
-
-object Address {
-  implicit val format: OFormat[Address] = Json.format[Address]
-}
+case class BusinessAddressModel(
+                                 auditRef: String,
```

Comment:
as I understand this field (auditRef), is used only in scope of the tests, so I would question if its really something required?